packages/contracts-bedrock/test/ExtendedPause.t.sol

```diff
@@ -4,24 +4,37 @@ pragma solidity 0.8.15;
 import { CommonTest } from "test/setup/CommonTest.sol";
 import { SuperchainConfig } from "src/L1/SuperchainConfig.sol";
 
+/// @dev These tests are somewhat redundant with tests in the SuperchainConfig and other pausable contracts, however
+///      it is worthwhile to pull them into one location to ensure that the behavior is consistent.
 contract ExtendedPause_Test is CommonTest {
     /// @dev Tests that other contracts are paused when the superchain config is paused
-    ///      This test is somewhat redundant with tests in the SuperchainConfig and other pausable contracts, however
-    ///      it is worthwhile to pull them into one location to ensure that the behavior is consistent.
-    function test_pause_fullSystem_succeeds() external {
+    function test_pause_fullSystem_succeeds() public {
         assertFalse(superchainConfig.paused());
-        assertEq(l1CrossDomainMessenger.paused(), superchainConfig.paused());
 
         vm.prank(superchainConfig.guardian());
         superchainConfig.pause("identifier");
 
+        // validate the paused state
         assertTrue(superchainConfig.paused());
-        assertEq(l1CrossDomainMessenger.paused(), superchainConfig.paused());
-
+        assertTrue(optimismPortal.paused());
+        assertTrue(l1CrossDomainMessenger.paused());
         assertTrue(l1StandardBridge.paused());
-        assertEq(l1StandardBridge.paused(), superchainConfig.paused());
-
         assertTrue(l1ERC721Bridge.paused());
-        assertEq(l1ERC721Bridge.paused(), superchainConfig.paused());
```

Comment:
The test I originally wrote was overkill, it's sufficient to ensure that all `paused()` methods return true.